.changeset/sweet-lamps-cover.md

```diff
@@ -0,0 +1,12 @@
+---
+'@astrojs/tailwind': patch
+'@astrojs/markdoc': patch
+'@astrojs/netlify': patch
+'@astrojs/svelte': patch
+'@astrojs/image': patch
+'@astrojs/mdx': patch
+'@astrojs/vue': patch
+'astro': patch
```

Comment:
I don't think we need all of these. Most of the integrations use Vite as a dev dep only. I think we can keep `'astro': patch` itself.